standalone-tools/wss-scan.sh

```diff
@@ -0,0 +1,121 @@
+###### Information ############################################################################
+# Name:          wss-scan.sh
+# Maintainer:    ODFE Infra Team
+# Language:      Shell
+#
+# About:         This script is to scan the odfe distros for vulnerabilities and licenses
+#                It will scan the repositories and send the WhiteSource link to the mail 
+#                of the user. 
+#
+# Prerequisites: Need to set the Java Path
```

Comment:
This is very vague, please considering making it more detailed like:

1. Need to install java (version 8+ or something)
2. export JAVA_HOME env variable to point to the jdk installation path
3. Add JAVA_HOME to PATH variable and export it.